starter/app/build.gradle

```diff
@@ -31,58 +34,55 @@ android {
         returnDefaultValues = true
     }
 
-    //dataBinding {
-    //    enabled = true
-    //    enabledForTests = true
-    //}
+//    dataBinding {
+//        enabled = true
+//        enabledForTests = true
+//    }
     buildFeatures {
         dataBinding true
     }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+
     // App dependencies
     implementation "androidx.appcompat:appcompat:$appCompatVersion"
-    implementation "androidx.legacy:legacy-support-v4:$androidXLegacySupport"
-    implementation "androidx.annotation:annotation:$androidXAnnotations"
-
-    implementation "androidx.cardview:cardview:$cardVersion"
+//    implementation "androidx.legacy:legacy-support-v4:$androidXLegacySupport"
+//    implementation "androidx.annotation:annotation:$androidXAnnotations"
+    implementation "androidx.swiperefreshlayout:swiperefreshlayout:$swipeRefreshVersion"
+//    implementation "androidx.cardview:cardview:$cardVersion"
     implementation "com.google.android.material:material:$materialVersion"
-    implementation "androidx.recyclerview:recyclerview:$recyclerViewVersion"
+//    implementation "androidx.recyclerview:recyclerview:$recyclerViewVersion"
     implementation "androidx.constraintlayout:constraintlayout:$constraintVersion"
 
-    implementation 'com.google.code.gson:gson:2.8.5'
+//    implementation 'com.google.code.gson:gson:2.8.5'
 
-    // Architecture Components
-    //Navigation dependencies
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.0-rc1'
```

Comment:
@graffiti75 Do you see [the first change of this PR](https://github.com/udacity/nd940-android-kotlin-c4-starter/pull/13/files) is not required here ?

Reply:
New PR created: https://github.com/udacity/nd940-android-kotlin-c4-starter/pull/14